modules/server/src/graphqlRoutes.js

```diff
@@ -281,9 +281,19 @@ export const createSchemasFromConfigs = async ({
 		/**
 		 * Federated Network Search
 		 */
-		const { networkSchema } = await createSchemaFromNetworkConfig({
-			networkConfigs: configsFromFiles[ConfigProperties.NETWORK_AGGREGATION],
-		});
+		if (ENABLE_NETWORK_AGGREGATION) {
+			const { networkSchema } = await createSchemaFromNetworkConfig({
+				networkConfigs: configsFromFiles[ConfigProperties.NETWORK_AGGREGATION].map((config) => ({
+					...config,
+					/**
+					 * part of the gql schema is generated dynamically
+					 * in the case of the "file" field, the field name and type name are the same
+					 * it's more flexible to define it here as an additional property than to confuse functions further down the pipeline
+					 */
+					documentName: config.documentType,
+				})),
+			});
+		}
```

Comment:
as comment says, `documentName` and `documentType` end up being the same mostly based on how Arranger dynamically builds the query. It makes more sense to add this early in the configuration than to conditional flip back and forth further down the stack. Other functions keep their distinction on which field they're looking at, and it'll be trivial to have the `documentName` configurable if Arranger dynamic schema generation changes